Loop/Views/SimpleBolusView.swift

```diff
@@ -28,6 +28,16 @@ struct SimpleBolusView: View {
             set: { newValue in viewModel.manualGlucoseString = newValue }
         )
     }
+    
+    private var enteredBolusString: Binding<String> {
+        Binding(
+            get: { return viewModel.enteredBolusString },
+            set: { newValue in
+                viewModel.enteredBolusString = newValue
+                viewModel.didEditBolusAmount = true
```

Comment:
Is there a reason this would ever need to be set back to false?

Reply:
it will only get set to true. false is just the starting value.